acceptance/openstack/networking/v2/networking.go

```diff
@@ -63,13 +63,12 @@ func CreateSubnet(t *testing.T, client *gophercloud.ServiceClient, networkID str
 	subnetOctet := tools.RandomInt(1, 250)
 	subnetCIDR := fmt.Sprintf("192.168.%d.0/24", subnetOctet)
 	subnetGateway := fmt.Sprintf("192.168.%d.1", subnetOctet)
-	iFalse := false
 	createOpts := subnets.CreateOpts{
 		NetworkID:  networkID,
 		CIDR:       subnetCIDR,
 		IPVersion:  4,
 		Name:       subnetName,
-		EnableDHCP: &iFalse,
+		EnableDHCP: gophercloud.Disabled,
```

Comment:
@jrperritt Is this along the same lines of `MaybeString`? I found this to be a helpful both in terms of being a shortcut as well as readability ("DHCP is Disabled").